lib/Runtime/Library/WebAssemblyInstance.cpp

```diff
@@ -300,7 +300,14 @@ void WebAssemblyInstance::LoadImports(
                 AsmJsScriptFunction* func = AsmJsScriptFunction::FromVar(prop);
                 if (!wasmModule->GetWasmFunctionInfo(counter)->GetSignature()->IsEquivalent(func->GetSignature()))
                 {
-                    JavascriptError::ThrowWebAssemblyLinkError(ctx, WASMERR_SignatureMismatch);
+                    char16 temp[1024] = { 0 };
```

Comment:
Actually, right now we fail the CI because of a prefast warning.
I don't think we have any tests that actually prints this. At most I think we catch the exception. Then again, I don't know how interesting it is to have a test just printing an error message.

As for size, we just have to make sure swprintf_s correctly crops the message and we don't have any errors if it becomes too long.